spikeextractors/sortingextractor.py

```diff
@@ -18,7 +18,7 @@ def __init__(self):
         self._unit_properties = {}
         self._unit_features = {}
         self._sampling_frequency = None
-        self.id = np.random.randint(low=0, high=9223372036854775807)
+        self.id = np.random.randint(low=0, high=9223372036854775807, dtype='int64')
```

Comment:
why the change? What was the issue with this being an `int`?

Reply:
see #303